packages/devtools/src/getMonolithicCSSRules.test.ts

```diff
@@ -158,11 +158,12 @@ describe('getMonolithicCSSRules', () => {
     });
   });
 
-  it('group atomic css when class name selector has selector in front', () => {
+  it.only('group atomic css when class name selector has selector in front', () => {
```

Comment:
```suggestion
  it('group atomic css when class name selector has selector in front', () => {
```

Can you please install [`eslint-plugin-jest`](https://github.com/jest-community/eslint-plugin-jest) and set `"jest/no-focused-tests": "error"`? 🐱 (in a separate PR as I am doing the same mistake often)

Reply:
yep let me do that in a separate PR :)

Reply:
Here we go: #208